xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-message/src/main/java/org/xwiki/rendering/internal/macro/message/XWikiErrorMessageMacro.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.internal.macro.message;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.localization.ContextualLocalizationManager;
+
+/**
+ * Displays an error message.
+ *
+ * @version $Id$
+ * @since 16.4.0RC1
+ */
+@Component
+@Named("error")
+@Singleton
+public class XWikiErrorMessageMacro extends ErrorMessageMacro
+{
+    private static String iconPrettyNameKey = "rendering.macro.message.icon.alternative.error";
+    @Inject
+    private ContextualLocalizationManager l10n;
+
+    /**
+     * Create and initialize the descriptor of the macro.
+     */
+    public XWikiErrorMessageMacro()
+    {
+        super();
+        this.iconPrettyName = l10n.getTranslation(iconPrettyNameKey).toString();
```

Comment:
You should use `getTranslationPlain` here to get the translation as plain text.

Reply:
Updated in https://github.com/xwiki/xwiki-platform/commit/9e7759387dc99ee81b038d9712712fd62ab6ada7 :+1: